p2p_wallet/Scenes/Main/Send/Input/SendInputViewModel.swift

```diff
@@ -149,6 +149,16 @@ final class SendInputViewModel: BaseViewModel, ObservableObject {
         initialize()
         logOpen()
         bind()
+        
+        // disable adding amount if amount is pre-chosen
```

Comment:
should we add an additional param for locking form? seems like there can be cases when we need pre-defined amount, but available to change it

Reply:
The form is already locked with isDisabled. Or I didn't catch it?